grass7/raster/r.centroids/r.centroids.html

```diff
@@ -0,0 +1,47 @@
+<h2>DESCRIPTION</h2>
+
+<em>r.centroids</em> is a wrapper function for <a href="https://grass.osgeo.org/grass-stable/manuals/r.volume.html">r.volume</a>
+that computes the center point of raster clumps of data (neighboring, same-valued pixels). The centroids will always fall within
+the clump so they may not be the true, mathematical centroid. The centroids will always fall at the center of a pixel.
+
+
+<h3>Options and flags:</h3>
```

Comment:
Sure. So it would look like this?

``<p>
<em>r.centroids</em> is a wrapper function for <a href="https://grass.osgeo.org/grass-stable/manuals/r.volume.html">r.volume</a>
that computes the center point of raster clumps of data (neighboring, same-valued pixels). The centroids will always fall within
the clump so they may not be the true, mathematical centroid. The centroids will always fall at the center of a pixel.
<p>"

Reply:
I think what @wenzeslaus meant was keep the `<h2>DESCRIPTION</h2>` heading and replace `<h3>Options and flags:</h3>` with `<p>`

Reply:
Ah. I will fix it to reflect that.

Reply:
Sorry, I actually meant to remove the heading _Options and flags_ completely. There no need for a subheading in a section with two paragraphs and three sentences total, so I wanted just an extra `<p>`. Definitively, `<h2>DESCRIPTION</h2>` should be kept there.

Reply:
Does what I said make sense? It seems to me that two headings for two paragraphs is unnecessary. There is also no need for `<p>Options and flags:</p>` since that section is auto-generated and there is no need here to go into details in another dedicated section.